src/pathways/escher.service.spec.ts

```diff
@@ -0,0 +1,90 @@
+// import {} from 'jasmine';
+
+import { EscherService } from './escher.service';
+
+describe('EscherService.alignReactions', () => {
```

Comment:
So this is just a human-readable test description prefix, which is displayed when a test fails. What do you mean that text search won't show it? Searching for `escher service` should still work and by convention, escher service tests will be in the corresponding `escher.service.spec.ts` file.